pandas/_libs/testing.pyx

```diff
@@ -210,4 +212,14 @@ cpdef assert_almost_equal(a, b,
                            f"with rtol={rtol}, atol={atol}")
         return True
 
+    if is_complex(a) and is_complex(b):
+        if array_equivalent(a, b, strict_nan=True):
+            # inf comparison
+            return True
+
+        if not cmath.isclose(a, b, rel_tol=rtol, abs_tol=atol):
```

Comment:
```suggestion
        if cmath.isclose(a, b, rel_tol=rtol, abs_tol=atol):
```

I think should just test for True here and return accordingly, otherwise let fall to the standard assertionerror, unless there is a reason for things to be different here?

Reply:
I copied what we do for reals, where we cite the `rtol` and `atol` if the comparison fails:
https://github.com/pandas-dev/pandas/blob/da1c6defda684ce1e006d4eac0035281b8cf3d4c/pandas/_libs/testing.pyx#L208-L210
whereas the message at the bottom is just:
https://github.com/pandas-dev/pandas/blob/da1c6defda684ce1e006d4eac0035281b8cf3d4c/pandas/_libs/testing.pyx#L213